servicebox-app/src/test/java/io/halkyon/CredentialsPageTest.java

```diff
@@ -20,7 +19,6 @@ public class CredentialsPageTest {
     WebPageExtension.PageManager page;
 
     @Test
-    @DisabledOnIntegrationTest
```

Comment:
No, because I deleted the `CredentialsPageIT.java` (before the application in native was being started for CredentialsPageIT when there were no tests (because the only one was disabled)